prebid-mobile/code-integration-android.md

```diff
@@ -38,20 +38,21 @@ to output the PrebidMobile framework for Android.
 
 Register Prebid Mobile ad units as early as possible in the application's lifecycle. Each ad unit has an `adUnitId` which is an arbitrary unique identifier of the developer's choice.
 
-The steps for using Prebid Mobile are as following:
+The steps for using Prebid Mobile are as follows:
 
 1. Create the ad units with ad unit ids and add sizes for banner ad units.
-2. Add a server side configuration for each ad unit to Prebid Server Adapter.
+2. Add a server-side configuration for each ad unit to Prebid Server Adapter.
 3. Set targeting parameters for the ad units. (Optional)
-4. Set the primary adserver for the bid to either DFP or MoPub (the primary ad server is necessary to determine the caching mechanism).
-5. Set the host for the bid. Supported hosts are AppNexus or Rubicon.
+4. Set the primary adserver for the bid to either DFP or MoPub. (Primary ad server is necessary to determine the caching mechanism.)
+5. Set the Host for the bid to AppNexus or Rubicon.
```

Comment:
Are these the only hosts that are currently supported?  It feels like this should point to list of supported PBS hosts that is separately maintained.